travis/shared_configs/package-linter.yml

```diff
@@ -0,0 +1,12 @@
+jobs:
+  include:
+    - stage: test
+      name: "Package Linter"
+      python: 3.6
+      install:
+        - pip install --upgrade pip
+        - pip install pcdsutils
+      before_script:
+        - cd "$TRAVIS_BUILD_DIR"
```

Comment:
I'm not sure... The `python-linter` template does the same so I kept the same behavior to make sure!

Reply:
[Looks like it does](https://travis-ci.com/github/hhslepicka/testproject1234/jobs/344643456#L203), but since `TRAVIS_BUILD_DIR` is an absolute path, it doesn't hurt to run it again.